### PR TITLE
fixed admin notice and full address shortcode issue

### PR DESCRIPTION
**Pull request by manishsubachchanyadav**
- Changed admin notices
- Fixed full address Shortcode

### CODE REVIEW

**Review by rajanvijayan — APPROVED** [review]
Looks good for me

**Comment by manishsubachchanyadav** [comment]
Fixed Gutenberg Elements 👍 
- Social Links with icon
- Logo Images

**Comment by manishsubachchanyadav** [comment]
Changed plugin version to v1.5.2

File: build/gutenberg-block.js [hunk 1]
```diff
@@ -1 +1,154 @@
...
+                    return parsedUrl.hostname === 'media-prod.apigateway.co';
+                } catch (e) {
+                    return false;
+                }
+            };
+
+            const getSocialIcon = (url) => {
+                try {
+                    const parsedUrl = new URL(url);
+                    if (parsedUrl.hostname.includes("facebook.com")) {
```
Comment (Padminia) [inline build/gutenberg-block.js]: Can you use a switch case here?

File: build/gutenberg-block.js [hunk 2]
```diff
@@ -1 +1,154 @@
...
```
Comment (Padminia) [inline build/gutenberg-block.js]: Can you use a switch case here as well?

**Review by Padminia — CHANGES_REQUESTED** [review]

**Comment by manishsubachchanyadav** [comment]
@Padminia Replaced the try-catch block with a switch-case statement for handling social links.

File: build/gutenberg-block.js (hunk 2 above)
Comment (manishsubachchanyadav) [reply build/gutenberg-block.js]: @Padminia Replaced the try-catch block with a switch-case statement for handling social links.

File: build/gutenberg-block.js (hunk 1 above)
Comment (manishsubachchanyadav) [reply build/gutenberg-block.js]: @Padminia Replaced the try-catch block with a switch-case statement for handling social links.

File: build/gutenberg-block.js [hunk 3]
```diff
@@ -1 +1,160 @@
...
+            const getSocialIcon = (url) => {
+                let hostname;
+                try {
+                    const parsedUrl = new URL(url);
+                    hostname = parsedUrl.hostname;
+                } catch (_) {
+                    return null;
+                }
+
+                switch (true) {
```
Comment (Padminia) [inline build/gutenberg-block.js]: Can you reuse the code by having it in the one function?

**Review by Padminia — CHANGES_REQUESTED** [review]

File: build/gutenberg-block.js (hunk 3 above)
Comment (manishsubachchanyadav) [reply build/gutenberg-block.js]: Can you check again ?

**Review by Padminia — APPROVED** [review]
LGTM

File: build/gutenberg-block.js
Comment (rajanvijayan) [inline build/gutenberg-block.js:1]: build files should be generated using npm cmd. You can edit only assets files
Please refer package.json 
```
"scripts": {
      "build:scripts": "npx wp-scripts build assets/js/gutenberg-block.js --output-path=build",
      "build:start": "npx wp-scripts start assets/js/gutenberg-block.js --output-path=build"
  },
```

File: src/Shortcode/Shortcode.php [hunk 4]
```diff
@@ -29,21 +29,19 @@ public static function render_business_profile($attr)
         // Get the attribute value from shortcode or use default 'company_name'
         $attr = isset($attr["attr"]) ? $attr["attr"] : BUSINESS_PROFILE_RENDER_DEFAULT_OPTION;
 
-
-        if (!array_key_exists($attr, $json_data)) {
-            return __("Attribute not found", "business-profile-render");
```
Comment (rajanvijayan) [inline src/Shortcode/Shortcode.php:34]: How can we handle this error? Can we move this part to the `else` case?

**Review by rajanvijayan — CHANGES_REQUESTED** [review]

File: src/Shortcode/Shortcode.php (hunk 4 above)
Comment (manishsubachchanyadav) [reply src/Shortcode/Shortcode.php:34]: - Generated by building files using cmd
- Used if else for Shortcode.php

**Review by rajanvijayan — APPROVED** [review]
Looks good for me

**Pull request merged**